### PR TITLE
Feature to add a comment

### PR DESCRIPTION
**Pull request by yonas44**
Hello @Abdessittir, so in this feature I implemented comment adding functionality, when you add a comment;
- When a user submits a comment through the form, it uses the involvement API to append a comment.

### CODE REVIEW

File: src/api/addComment.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const addComment = async (id, name, insight) => {
+  try {
+    const res = await fetch(`${process.env.INVOLVMENT_URL}/apps/${process.env.APP_ID}`, {
```
Comment (Abdessittir) [inline src/api/addComment.js:3]: Please make sure that the URL is correct

**Review by Abdessittir — CHANGES_REQUESTED** [review]
Hi @yonas44  great work 👍 , It was so close

# STATUS: CHANGES REQUIRED :recycle: 
soclose

Here are some things you need to work on:

 - [ ] Take a look at the comment below
 - [ ] Kindly make sure that the comment show up un the screen when it gets added, because that is included in the project requirement
 
req

commenttoDom

# OPTINAL:
Those are optional and you can skip them:

 - [ ] Kindly give the form elements some padding, so it look nice on the screen
 
padding

## If you have any questions please feel free to ask 👍 

Happy coding 💯

**Comment by yonas44** [comment]
Thank you @Abdessittir  for your feedback, I have added the changes please have a look at it and give me your feedback!

**Review by Abdessittir — APPROVED** [review]
Hi @yonas44 amazing work 💯, there is nothing to say other than it's time to merge your pull request.

# STATUS: APPROVED 🥇 

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# Highlights 💪🏻

✅ Comment posted to the API correctly
✅ No linter errors
✅ Dom manipulated to add new comment 💯 

Happy Coding 👍

**Pull request merged**